tests/check_consistent.py

```diff
@@ -128,7 +128,28 @@ def check_versions():
     assert not extra, f"Versions not in modules: {extra}"
 
 
+def _strip_dep_version(dependency):
+    dep_version_pos = len(dependency)
+    for pos, c in enumerate(dependency):
+        if c in "=<>":
```

Comment:
What if there is a semicolon? Example from mypy `test-requirements.txt`:
```
flake8-bugbear; python_version >= '3.5'
```

We check for spaces, but somebody might write something like this:

```
types-foo;python_version>='3.5'
```

Maybe use a regexp to validate the version string so that we don't check for bad things but ensure that only expected things exist?

Reply:
Again, there should not be semicolons, `requires` is already a list.

Reply:
Sorry, I didn't notice what is in the second part, I think we may not support this initially.